nopen/Makefile

```diff
@@ -0,0 +1,39 @@
+CC=			gcc
```

Comment:
You can `include ../config.mk` to get the settings from the top level `configure`